packages/wrangler/src/pipelines/cli/create.ts

```
@@ -29,274 +16,132 @@ export const pipelinesCreateCommand = createCommand({
+
+		let pipeline;
+		try {
+			pipeline = await createPipeline(config, pipelineConfig);
+		} catch (error) {
+			if (error instanceof APIError && error.code === 10000) {
+				// Show error when no access to v1 Pipelines API
+				throw new UserError(
+					`An authentication error has occurred creating a pipeline with this version of Wrangler. Please try: npm install wrangler@4.36.0`
+				);
+			}
+			throw error;
 		}
 
-		logger.log(`🌀 Creating pipeline named "${name}"`);
-		const pipeline = await createPipeline(config, accountId, pipelineConfig);
-
 		logger.log(
-			`✅ Successfully created pipeline "${pipeline.name}" with ID ${pipeline.id}\n`
+			`✨ Successfully created pipeline '${pipeline.name}' with id '${pipeline.id}'.`
 		);
-		logger.log(formatPipelinePretty(pipeline));
-		logger.log("🎉 You can now send data to your pipeline!");
 
-		if (args.source.includes("worker")) {
-			await updateConfigFile(
-				(bindingName) => ({
-					pipelines: [
-						{
-							pipeline: pipeline.name,
-							binding: getValidBindingName(
-								bindingName ?? "PIPELINE",
-								"PIPELINE"
-							),
-						},
-					],
-				}),
-				config.configPath,
-				args.env
+		try {
+			const fullPipeline = await getPipeline(config, pipeline.id);
+
+			const streamTable = fullPipeline.tables?.find(
+				(table) => table.type === "stream"
 			);
-		}
 
-		if (args.source.includes("http")) {
-			logger.log(`\nSend data to your pipeline's HTTP endpoint:\n`);
-			logger.log(`curl "${pipeline.endpoint}" -d '[{"foo": "bar"}]'\n`);
+			if (streamTable) {
+				const stream = await getStream(config, streamTable.id);
+				displayUsageExamples(stream);
+			} else {
+				logger.log(
+					`\nRun 'wrangler pipelines get ${pipeline.id}' to view full details.`
+				);
+			}
+		} catch {
+			logger.warn("Could not fetch pipeline details for usage examples.");
+			logger.log(
+				`\nRun 'wrangler pipelines get ${pipeline.id}' to view full details.`
+			);
 		}
```

Why is this error being suppressed?

---

Don't want to fail command  in this case